spacy/language.py

```diff
@@ -157,7 +156,7 @@ class Language(object):
     }
 
     def __init__(
-        self, vocab=True, make_doc=True, max_length=10 ** 6, meta={}, **kwargs
+        self, vocab=True, make_doc=True, max_length=10 ** 6, meta=None, **kwargs
```

Comment:
I'm slightly worried that this could have unintended side-effects. Reconciling the meta defined in the language defaults and the meta passed in by the user could also end up being problematic and lead to unexpected results. So I'm not sure language subclasses should be allowed to define the full meta.

I do see the point of adding the requirements, though. (If a user saves out the `nlp` object and packages the model, the Python package will include these dependencies in its setup.) So an alternative solution could be to introduce a `requirements` attribute on the `Defaults` and then put this together in the `Language.meta` property?

Reply:
Agree that `requirements` seems like a good idea. Is there a way to avoid the duplication though? Can we populate it from `importlib_meta` using the language code?

Reply:
Ahh, that's a clever idea! And yes, I think this should be possible. I'll look into this! 👍